iati_dashboard/ui/views.py

```diff
@@ -543,6 +543,41 @@ def exploringdata_codelists_detail(request, major_version=None, attribute=None):
     return HttpResponse(template.render(context, request))
 
 
+def exploringdata_publisher_codelist_detail(request, publisher_short_name=None, major_version=None, attribute=None):
+    try:
+        publisher = models.ReportingOrg.objects.get(short_name=publisher_short_name)
+    except models.ReportingOrg.DoesNotExist:
+        raise Http404("Publisher does not exist")
+
+    attribute = ".//" + attribute.replace("_", "/")
+    if attribute.endswith("/text"):
+        attribute += "()"
+    values = publisher.stats_json.get("codelist_values_by_major_version", {}).get(major_version, {}).get(attribute)
+
+    context = _make_context("publishers", include_large_dicts=False)
+    context["publisher"] = publisher
+    context["element"] = attribute
+    context["major_version"] = major_version
+    context["values"] = values
+    # FIXME this is duplicated above
```

Comment:
Should these instances of `FIXME` be addressed as part of this? Or is this logged in a GH Issue somewhere?